lib/cdo/aws/cloudfront.rb

```diff
@@ -227,7 +227,8 @@ def self.distribution_config(app, origin, aliases, ssl_cert = nil)
           CloudFrontDefaultCertificate: true,
           MinimumProtocolVersion: 'TLSv1' # accepts SSLv3, TLSv1
         },
-        HttpVersion: 'http2'
+        HttpVersion: 'http2',
+        WebACLId: {'Fn::Sub': 'arn:aws:wafv2:${AWS::Region}:${AWS::AccountId}:global/webacl/code-dot-org/a79a67de-dabf-4555-a8dc-98c3a20b562f'}
```

Comment:
Will this apply to adhocs?  E.g. will it put the bugcrowd adhoc behind the prod WAF?

Reply:
It would. There isn't a different ACL existing today that we would want to use for adhocs, so the question is whether they should be added to the ACL or have no ACL.

Reply:
Hmm, @sureshc did we want to avoid associating the Test distributions, because of how UI tests might look like DDOS attacks?

Reply:
sorry for the slow response. We have protected every environment that’s in our build pipeline from attacks with a single Web Application Firewall Access Control List (WAF ACL). I’m fine with excluding `adhoc`, but I think it’s worth including `staging`, `test`, and `levelbuilder`.

Reply:
I'm fine including everything for this iteration, and then we can add exclusions if we get false positives in Test, or find it's annoying for adhocs for some reason.